tooling/nargo_cli/benches/criterion.rs

```diff
@@ -23,6 +23,8 @@ fn compile_program(test_program_dir: &Path) {
     cmd.arg("--program-dir").arg(test_program_dir);
     cmd.arg("compile");
     cmd.arg("--force");
+    cmd.arg("--expression-width");
+    cmd.arg("1");
```

Comment:
What does this "1" mean here?

Reply:
I don't think this should be here right?

Reply:
Good cat, I think this is what I used to trigger the error and accidentally checked it in, thanks!